developers_chamber/qa/checks.py

```diff
@@ -0,0 +1,78 @@
+import re
+
+from .base import QACheck, QAError
+
+
+class MissingMigrationsQACheck(QACheck):
+    """
+    Checks that make migrations command does not generate new migrations.
+    """
+    name = 'Check missing migrations'
+
+    def _run_check(self):
+        output = self._run_command('pydev make-migrations --dry-run')
+        last_line = output.split('\n')[-1]
+        if last_line != 'No changes detected':
+            raise QAError('Found missing migration(s)!', re.findall(r'\x1b.*?Migrations.+$', output, re.DOTALL)[0])
+
+
+class MigrationFilenamesQACheck(QACheck):
```

Comment:
It should happen for our projects now?

Reply:
Well, I implemented this in times when the "only-numbers-in-migration-filenames" policy was in effect. Now, it's usefulness is limited, because migrations are automatically generated with correct name, but I would keep it, in case someone introduces a manual migration and forgets to name it correctly.